proposals/proposal_1/lib/src/settings/app_theme.dart

```diff
@@ -0,0 +1 @@
+enum AppTheme { light, dark }
```

Comment:
Isn't this equivalent to https://api.flutter.dev/flutter/material/ThemeMode-class.html?

Reply:
It's a fair point -- I wanted this to be "in the control of the developer" in case they wanted to add additional, custom themes as well -- think Slack with lots of theme options.

However, the naming is bad here. I'll look at how to simplify this, or if we should just store the ThemeMode.

Reply:
Switched it up to ThemeMode for now! Thanks again for the suggestion :)